master/buildbot/master.py

```diff
@@ -228,7 +238,7 @@ def startService(self):
                 # run the master.cfg in thread, so that it can use blocking code
                 self.config = yield threads.deferToThreadPool(
```

Comment:
I think we should put the deferToThread in FileLoader.

I would *love* to be able to write asynchronous config_loader